sale_order_carrier_auto_assign/__manifest__.py

```diff
@@ -7,7 +7,8 @@
     "development_status": "Beta",
     "category": "Operations/Inventory/Delivery",
     "website": "https://github.com/OCA/sale-workflow",
-    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "author": "Camptocamp, BCIM, Odoo Community Association (OCA)",
```

Comment:
I'm curious. Where did you enter that copyrights? I doubt my colleague removes them intentionally.